src/common/wal/src/config.rs

```diff
@@ -17,16 +17,16 @@ pub mod raft_engine;
 
 use serde::{Deserialize, Serialize};
 
-use crate::config::kafka::{DatanodeKafkaConfig, MetaSrvKafkaConfig, StandaloneKafkaConfig};
+use crate::config::kafka::{DatanodeKafkaConfig, MetasrvKafkaConfig, StandaloneKafkaConfig};
 use crate::config::raft_engine::RaftEngineConfig;
 
 /// Wal configurations for metasrv.
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Default)]
 #[serde(tag = "provider", rename_all = "snake_case")]
-pub enum MetaSrvWalConfig {
+pub enum MetasrvWalConfig {
```

Comment:
Will there be any issue if I make this change? cc @niebayes

Reply:
> Will there be any issue if I make this change? cc @niebayes

Looks like it will break existing config files. But luckily we don't actual have instance with this config.

Reply:
> Will there be any issue if I make this change? cc @niebayes

Looks like it will break existing config files. But luckily we don't actual have instance with this config.